pkg/asset/agent/image/oc.go

```diff
@@ -64,7 +64,7 @@ const (
 )
 
 // Get the CoreOS ISO from the releaseImage
-func (r *release) GetBaseIso(log logrus.FieldLogger, releaseImage, pullSecret, architecture string, mirrorConfig []mirror.RegistriesConfig) (string, error) {
+func (r *release) GetBaseIso(log logrus.FieldLogger, releaseImage, pullSecret, rhcosArch string, mirrorConfig []mirror.RegistriesConfig) (string, error) {
```

Comment:
Why rename this when the field that is passed in (amd64/arm64) isn't even in the format used by rhcos (x86_64/aarch64)?
We should define some constants and give them separate types so that we can't mix the two up.